blog/2024-10-30-openrefine-developer-role.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Martin Magdinier
+title: "Now Hiring: OpenRefine Developer"
+---
+## About the role 
+
+* Role: Software Developer
+* Job Type: Part-Time Contract
+* Duration: 9 months with possible extension.
+* Job Location: Remote
+* Total funding available for the position: USD 90,000 annually 
+
+## Role Summary
+
+This part-time role for an OpenRefine Developer, contracted through [Code for Science & Society Inc.](https://www.codeforsociety.org/) (a 501(c)(3) charitable organization in the USA), will focus on enhancing OpenRefine's stability, reliability, and quality to support the project’s long-term sustainability. This role is funded by the [EOSS-5 grant](/funding#2022-eoss-5).
+
+Key responsibilities include bug fixes, quality improvements, and ongoing maintenance, with an emphasis on features supported by grant funding. Over time, the developer will expand into release management, security maintenance, and mentorship within internship programs (e.g., GSoC, Outreachy).
+
+## Core Responsibilities
+* **Bug Fixes & Quality Improvements**
+  * Prioritize and address bugs within the codebase to enhance stability and performance.
+  * Improve reproducibility features (per EOSS-5 grant requirements).
+  * Implement quality improvements through refactoring, optimization, and code maintenance.
+* **Community Support & Engagement**
+  * Participate in community forums to support OpenRefine users and volunteers, fostering a positive and engaged user community.
+* **Pull Request Reviews & Ticket Triage**
+  * Once familiar with the codebase, actively engage with community-submitted pull requests, providing constructive feedback and participating in collaborative review processes.
+  * Assist with ticket triage to identify and prioritize key issues and opportunities.
```

Comment:
```suggestion
  * Assist with issue triage to identify and prioritize project goals and opportunities.
```

Reply:
We should always be triaging with an eye towards project goals or goal posts or milestones, as well as security issues.